src/app/settings/page.tsx

```diff
@@ -35,31 +37,21 @@ type FormValues = {
   phoneNumber: string;
   gender: (typeof GENDERS)[number];
   industry: (typeof INDUSTRIES)[number] | '';
-  // role: (typeof ROLES)[number];
-  // permissions: (typeof PERMISSIONS)[number];
 };
 
 export default function ProfilePage() {
-  // TODO: get user information from the session - using example data for now
-  const currentUser: FormValues = {
-    firstName: 'John',
-    lastName: 'Doe',
-    email: 'johndoe@email.com',
-    phoneNumber: '+1 (123) 456-7890',
-    gender: 'Male',
-    industry: 'Technology',
-  };
+  const { user } = useAuth();
+  const [currentUser, setCurrentUser] = useState<FormValues | null>(null);
+  const [loading, setLoading] = useState(true);
```

Comment:
should probably have an error state so that we can display an error page if there is one rather than just log it if we can't get the user info. doesn't have to be implemented now but we should take note of it for other pages too